advanced/acronym.rb

```diff
@@ -1,3 +1,16 @@
 class Acronym
+#Convert a phrase to its acronym.
+
+# Techies love their TLA (Three Letter Acronyms)!
+
+# Help generate some jargon by writing a program that converts a long name like Portable Network Graphics to its acronym (PNG).
+ 
+# Your Job
+# Define a class called Acronym with a class method called abbreviate that accepts one String argument, and returns a String.
+
+# Example
+
+# Acronym.abbreviate("Portable Network Graphics") # => "PNG"
+
```

Comment:
I do think I like this format! It's not as overwhelming as I was imagining. Maybe it will also make them more comfortable with comments.